Decoupled testdata plugin (#72205)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

| `grafanaAPIServer`                          | Enable Kubernetes API Server for Grafana resources                                                                                                                                       |
| `featureToggleAdminPage`                    | Enable admin page for managing feature toggles from the Grafana front-end                                                                                                                |
| `awsAsyncQueryCaching`                      | Enable caching for async queries for Redshift and Athena. Requires that the `useCachingService` feature toggle is enabled and the datasource has caching and async query support enabled |

> | `decoupleCorePlugins`                       | Skip loading core plugins on startup in order to load them as external                                                                                                                   |

Reviewer: ```suggestion
| `decoupleCorePlugins`                       | To load core plugins externally, this setting bypasses loading them during startup.                                                                                                                   |
```
